tests/test_admin_filters.py

```diff
@@ -166,3 +168,51 @@ def test_site_filter(self):
             set(response_empty_site.context["cl"].queryset),
             set([])
         )
+
+
+@skipUnless(is_versioning_enabled(), 'Test only relevant for versioning')
+class UnpublishedFiltersTestCase(BaseAliasPluginTestCase):
+
+    def test_unpublished_filter(self):
```

Comment:
Could you make this test follow AAA? 

https://jamescooke.info/arrange-act-assert-pattern-for-python-developers.html

Reply:
Updated